edb/edgeql/utils.py

```diff
@@ -119,3 +120,27 @@ def inline_anchors(
 
     inliner = AnchorInliner(anchors)
     inliner.visit(ql_expr)
+
+
+MAX_INT64 = 2 ** 63 - 1
+MIN_INT64 = -2 ** 63
+
+
+def const_ast_from_python(val: Any) -> qlast.BaseConstant:
+    if isinstance(val, str):
+        return qlast.StringConstant.from_python(val)
+    elif isinstance(val, bool):
+        return qlast.BooleanConstant(value='true' if val else 'false')
+    elif isinstance(val, int):
+        if MIN_INT64 <= val <= MAX_INT64:
```

Comment:
I thought we decided against this, no?

Reply:
It's literally illegal then to create that `IntegerConstant` for cases outside the range. Why is it the only one that we're OK with messing up? The place where the utility function is used is generic, so technically there could be a large integer there.

Reply:
I moved it to a different utils - in `schema`, closer to `delta.py`